air/src/execution.rs

```diff
@@ -0,0 +1,23 @@
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct ExecutionOptions {
+    max_cycles: Option<u32>,
+    expected_cycles: u32,
+}
```

Comment:
I would probably put this into `options.rs` file and also move `ProvingOptions` there.

Also, since this is a part of the public interface, we should add doc comments to the struct itself and its public methods.